package.json

```diff
@@ -44,11 +44,14 @@
     "url": "https://github.com/UCSF-IGHS/openmrs-esm-rwanda/issues"
   },
   "dependencies": {
-    "@carbon/react": "^1.33.1",
+    "@carbon/react": "1.37.0",
```

Comment:
This change can be reverted. Not a blocker though cause it's being overriden by the resolution

Reply:
Roger that